test/configCases/records/issue-2991/webpack.config.js

```diff
@@ -0,0 +1,26 @@
+var path = require("path");
+
+module.exports = {
+	entry: "./test",
+	recordsPath: path.resolve(__dirname, "../../../js/config/records/issue-2991/records.json"),
+	target: "node",
+	node: {
+		__dirname: false
+	},
+	plugins: [
+		{
+			apply(compiler) {
+				compiler.plugin("normal-module-factory", (nmf) => {
```

Comment:
i tried to, however as the IgnoredPlugin intercepts on 'before-resolve' it never gets passed this point: https://github.com/webpack/webpack/blob/master/lib/NormalModuleFactory.js#L235

therefore never makes it into the factory plugin and does not create a `ignored` entry.

The question of course is if it should do that, however there is no real benefit in it is there?

maybe more general, is there even a need to record ignored modules at all?